packages/hardhat-core/src/internal/hardhat-network/provider/utils/makeFakeSignature.ts

```diff
@@ -0,0 +1,23 @@
+import crypto from "crypto";
+import util from "util";
+
+// Produces a signature with r and s values taken from a hash of the inputs.
+export function makeFakeSignature(...inputs: any[]): {
+  v: number;
+  r: number;
+  s: number;
+} {
+  const hash = crypto.createHash("md5");
```

Comment:
https://github.com/NomicFoundation/hardhat/blob/main/packages/hardhat-core/src/internal/util/hash.ts#L10

Reply:
I updated this to use the existing function, in https://github.com/NomicFoundation/hardhat/pull/3175/commits/9d048a68a794f745dea8d10828b0a5787b7eff19